frontend/model/contracts/group.js

```diff
@@ -1434,6 +1444,13 @@ sbp('chelonia/defineContract', {
 
       sbp('okTurtles.events/emit', PAYMENTS_ARCHIVED, { paymentsByPeriod, payments })
     },
+    'gi.contracts/group/removeArchivedPayments': async function (contractID) {
+      const { username } = sbp('state/vuex/state').loggedIn
+      const archPaymentsByPeriodKey = `paymentsByPeriod/${username}/${contractID}`
+      const archSentOrReceivedPaymentsKey = `sentOrReceivedPayments/${username}/${contractID}`
+      await sbp('gi.db/archive/delete', archPaymentsByPeriodKey)
+      await sbp('gi.db/archive/delete', archSentOrReceivedPaymentsKey)
+    },
```

Comment:
Nice, but you forgot `payments/${username}/${period}/${contractID}`

Reply:
Oh, yes. Nice catch. 👍🏼